index.html

```diff
@@ -0,0 +1,77 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio page</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.2/css/all.min.css" integrity="sha512-z3gLpd7yknf1YoNbCzqRKc4qyor8gaKU1qmn+CShxbuBusANI9QpRohGBreCFkKxLhei6S9CQXFEbbKuqLg0DA==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+</head>
+<body>
+    <nav>
+        <div class="logo">
+      <p class="log" >portfo<span>lio</span></p>  
+        </div>
+        <ul>
+        <li><a href="#">Home</a></li> 
+        <li> <a href="#">About</a></li>
+        <li><a href="#">Skills</a></li>
+        <li><a href="#">Projects</a></li>
+        <li><a href="#">Contact</a></li>
+        </ul>
+    </nav>
+
+    <header>
+        <h1 class="name">Halima Hassan</h1>
+        <p>Front End Developer</p> 
+    </header> 
+
+    <main>
+        <section class="about">
+          <h4>Hello,my name is</h4>
+          <h1>Halima Hassan,</h1>
+          <h3>a skilled Front-end Developer specializing in HTML, CSS, and JavaScript. I have a knack for turning designs into immersive, visually appealing websites that engage and delight users. Let's bring your ideas to life through code and creativity!
+          </h3>
+        </section>
+
+        <section class="skills">
+            <h2>Skills</h2>
+            <ul>
+                <li>Html</li>
+                <li>CSS</li>
+                <li>java Script</li>
+                <li>java</li>
+            </ul>
+            </section> 
+
+            <section class="contact-info">
+                <h2>Contact information</h2>
+                <ul>
+                    <li>
+                        <i class="fa fa-linkedin" aria-hidden="true"></i>
+                       <a href="https://www.linkedin.com/feed/">Email</a>
+                   </li>
+                <li>
+                     <i class="fa fa-linkedin" aria-hidden="true"></i>
+                    <a href="https://www.linkedin.com/feed/">Linkdin</a>
+                </li>
+                
+                <li>
+                    <i class="fa fa-linkedin" aria-hidden="true"></i>
+                    <a href="https://github.com/halima770"> github</a>
+                </li>
+                
+                </ul>
+              </section>
+     
+              <section id="portfolio">

```

Comment:
Why section has 'id' here while the other sections have 'class' ?  I recommend avoiding using 'id' as the best practice is using 'class'